centreon/www/class/config-generate/AdditionalConnectorVmWareV6.class.php

```diff
@@ -101,4 +101,30 @@ public function generateFromPollerId(int $pollerId): void
     {
         $this->generate($pollerId);
     }
+
+    /**
+     * @param $dir
+     *
+     * @throws \RuntimeException|\Exception
+     */
+    protected function writeFile($dir)
+    {
+        $full_file = $dir . '/' . $this->generate_filename;
```

Comment:
Try to follow naming conventions: **CamelCase**
```suggestion
        $fullFile = $dir . '/' . $this->generate_filename;
```